frontend/src/metabase-types/api/data-app.ts

```diff
@@ -48,6 +49,7 @@ export interface ActionDashboardCard
   extends Omit<BaseDashboardOrderedCard, "parameter_mappings"> {
   action?: WritebackAction;
   card_id?: number; // model card id for the associated action
+  card?: Card;
```

Comment:
What type of cards get linked to `ActionDashboardCard`? Is it the card backing the query action object? If so, it might be dangerous to use it. Once we introduce HTTP actions that don't have a backing card, it would make us maintain two different ways of doing things against action dash cards

Reply:
it's the card backing the model, so, it should stay stable between query and http actions. If actions get divorced from models though, this will 💥